manifests/kfdef/odh-dashboard-kfnbc-test.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: kfdef.apps.kubeflow.org/v1
+kind: KfDef
+metadata:
+  annotations:
+    kfctl.kubeflow.io/force-delete: "false"
+  name: odh-dashboard-kfnbc-test
```

Comment:
When using this kfdef I don't see the notebook controller card in the "Enabled" tab:

![image](https://user-images.githubusercontent.com/27858594/178467445-306aaf8f-5b81-45b6-93a2-f3222a48fbab.png)

What are the instructions to enable it?

Reply:
There are some missing configurations to support the latest dashboard updates for KFNBC and this looks like one of them.

From you comment in the KFNBC UI PR https://github.com/opendatahub-io/odh-dashboard/pull/267#pullrequestreview-1032628126, it looks like I need to add to the manifest

```
kind: ConfigMap
apiVersion: v1
metadata:
  name: odh-enabled-applications-config
data:
  jupyter: 'true'
  jupyterhub: 'true'
```